src/azul/openapi/schema.py

```diff
@@ -338,7 +338,7 @@ def in_range(minimum: Optional[N], maximum: Optional[N], type_: Optional[TYPE] =
 }
 
 
-def object_type(properties: JSON, **kwargs) -> JSON:
+def object_type(properties: JSON, additional_properties=False, **kwargs) -> JSON:
```

Comment:
This goes against the philosophy behind the `…_type` functions. They don't treat any schema properties special by elevating them to keyword arguments. Also note the call to this in line 93 that still passes this in kwargs rather than the special keyword argument. Also, while you're touching this: `properties` should be a positional only argument.

So

```suggestion
def object_type(properties: JSON, /, **kwargs) -> JSON:
```